docs/source/deployment/aws_ec2.rst

```diff
@@ -0,0 +1,370 @@
+
+* AWS SAM CLI tool
+
+  * Install instruction: https://docs.aws.amazon.com/serverless-application-model/latest/developerguide/serverless-sam-cli-install.html
+
+* Docker is installed and running on the machine.
+
+  * Install instruction: https://docs.docker.com/install
+
+
+AWS EC2 deployment and management with BentoML
+-------------------------------------------------
+
+This guide uses the IrisClassifier BentoService from the :doc:`Quick start guide <../quickstart>`:
+
+.. code-block:: bash
+
+    git clone git@github.com:bentoml/BentoML.git
+    pip install -r ./bentoml/guides/quick-start/requirements.txt
+    python ./bentoml/guides/quick-start/main.py
+
+Use `bento list` to get the BentoService name:version tag.
+
+
+.. code-block:: bash
+
+    > bentoml list
+
+    BentoService                          CREATED_AT                  APIS                        ARTIFACTS
+    IrisClassifier:20200121141808_FE78B5  2020-01-21 22:18:25.079723  predict(DataframeInput)  model(SklearnModelArtifact)
+
+
+BentoML has great support for AWS EC2. You can deploy, update and delete
+deployment with single command, and customize deployment to fit your needs with parameters
+such as `instance type`,`scaling capacities`
+
+To deploy BentoService to AWS EC2, use `bentoml lambda deploy` command.
+Deployment name and bento service name:version tag is required.
+
+.. code-block:: bash
+
+    > bentoml ec2 deploy my-first-ec2-deployment -b IrisClassifier:20200121141808_FE78B5
+
+    Deploying EC2 deployment -[2020-01-21 14:37:16,838] INFO - Building project
+    [2020-01-21 14:38:52,826] INFO - Containerzing service
+    [2020-01-21 14:39:18,834] INFO - Deploying
+    [2020-01-21 14:40:09,265] INFO - ApplyDeployment (my-first-ec2-deployment, namespace dev) succeeded
+    Successfully created AWS EC2 deployment my-first-ec2-deployment
+    {
+    "namespace": "dev",
+    "name": "my-first-ec2-deployment",
+    "spec": {
+        "bentoName": "IrisClassifier",
+        "bentoVersion": "20200121141808_FE78B5",
+        "operator": "AWS_EC2",
+        "awsEc2OperatorConfig": {
+        "region": "ap-south-1",
+        "instanceType": "t2.micro",
+        "amiId": "/aws/service/ami-amazon-linux-latest/amzn2-ami-hvm-x86_64-gp2",
+        "autoscaleMinCapacity": 1,
+        "autoscaleDesiredCapacity": 1,
+        "autoscaleMaxCapacity": 1
+        }
+    },
+    "state": {},
+    "createdAt": "2020-10-24T06:52:18.003580Z",
+    "lastUpdatedAt": "2020-10-24T06:52:18.003626Z"
+    }
+
+
+
+BentoML helps you manage the entire process of deploying your BentoService bundle to EC2.
+Verify the deployed resources with AWS CLI tool:
+
+.. code-block:: bash
+
+    > aws cloudformation describe-stacks
+
+    {
+        "Stacks": [
+            {
+                "StackId": "arn:aws:cloudformation:us-west-2:192023623294:stack/dev-my-first-lambda-deployment/dd2a7cf0-3c9e-11ea-8654-02f6ffa9fe66",
```

Comment:
why it says "my-first-lambda-deployment" here in the name?  I guess it was an older message where you were using a different deployment name?